src/connectors/articleService.ts

```diff
@@ -316,7 +316,7 @@ export class ArticleService extends BaseService {
       })
       return
     }
-    const directoryName = `${kname}-with-${lastDraft.id}-${slugify(
+    const directoryName = `${kname}-with-${lastDraft.articleId}-${slugify(
```

Comment:
not only here, the fix will need to be in `publishedDrafts` all item's `id` will need to be `articleId`, and ipns-site-manager will need to change there?

Reply:
`@matters/ipns-site-generator` doesn't depend on our article or draft object from database, just render what we pass into it.

https://github.com/thematters/ipns-site-generator/blob/main/src/types.ts